apps/wing/src/cli.ts

```diff
@@ -123,7 +123,7 @@ async function main() {
     .command("run")
     .alias("it")
     .description("Runs a Wing program in the Wing Console")
-    .argument("[entrypoint]", "program .w entrypoint")
+    .argument("[entrypoint]", "program .w or .wsim entrypoint")
```

Comment:
I am not sure this is something we want to (or need to) support long term so I wouldn't advertise it.